### PR TITLE
Refactor IssueStandartMapper and IssueStandartEntity

### PR DESCRIPTION
**Pull request by janchk**
#248 
- [x] Refactor  IssueStandartMapper
- [x] Refactor IssueStandartEntity

### CODE REVIEW

File: src/test/java/easytests/core/mappers/IssueStandardsMapperTest.java [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 import easytests.core.entities.IssueStandardEntity;
 import java.util.List;
+
```
Comment (malinink) [inline src/test/java/easytests/core/mappers/IssueStandardsMapperTest.java]: remove empty line

File: src/test/java/easytests/core/mappers/IssueStandardsMapperTest.java [hunk 2]
```diff
@@ -12,102 +15,90 @@
...
 
     @Autowired
     private IssueStandardsMapper issueStandardsMapper;
 
+    protected IssueStandardSupport issueStandardSupport = new IssueStandardSupport();
+
     @Test
     public void testFindAll() throws Exception {
-        List<IssueStandardEntity> issueStandardEntities = this.issueStandardsMapper.findAll();
+        List<IssueStandardEntity> issueStandatrsFoundedEntities = this.issueStandardsMapper.findAll();
```
Comment (malinink) [inline src/test/java/easytests/core/mappers/IssueStandardsMapperTest.java]: standar**d**s

File: src/test/java/easytests/core/mappers/IssueStandardsMapperTest.java [hunk 3]
```diff
@@ -12,102 +15,90 @@
...
+
+        Assert.assertNotNull(issueStandatrsFoundedEntities);
```
Comment (malinink) [inline src/test/java/easytests/core/mappers/IssueStandardsMapperTest.java]: no need for check that in mapper test, it is mybatis responsibility

File: src/test/java/easytests/core/mappers/IssueStandardsMapperTest.java [hunk 4]
```diff
@@ -12,102 +15,90 @@
...
-        this.issueStandardsMapper.update(issueStandardEntity);
-
-        issueStandardEntity = this.issueStandardsMapper.find(id);
-        Assert.assertEquals(id, issueStandardEntity.getId());
-        Assert.assertEquals(timeLimit, issueStandardEntity.getTimeLimit());
-        Assert.assertEquals(questionsNumber, issueStandardEntity.getQuestionsNumber());
-        Assert.assertEquals(subjectId, issueStandardEntity.getSubjectId());
+        final IssueStandardEntity issueStandardChengedEntity = this.issueStandardSupport.getEntityAdditionalMock(1);
+
+        final Integer id = issueStandardChengedEntity.getId();
```
Comment (malinink) [inline src/test/java/easytests/core/mappers/IssueStandardsMapperTest.java]: refuse of using id (just for having similar code in all mappers tests)

File: src/test/java/easytests/support/IssueStandardSupport.java [hunk 5]
```diff
@@ -0,0 +1,153 @@
+package easytests.support;
+
+import easytests.core.entities.IssueStandardEntity;
+import easytests.core.models.IssueStandardModelInterface;
+import easytests.core.models.empty.ModelsListEmpty;
+import easytests.core.models.empty.SubjectModelEmpty;
+import org.junit.Assert;
+import org.mockito.Mockito;
+
```
Comment (malinink) [inline src/test/java/easytests/support/IssueStandardSupport.java]: add newline

File: src/test/java/easytests/support/IssueStandardSupport.java [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+            {
+                    // for update entity with id = 1
+                    1,
+                    3600,
+                    20,
+                    2
+            },
+    };
+
+    public IssueStandardEntity getEntityFixtureMock(Integer index){
```
Comment (malinink) [inline src/test/java/easytests/support/IssueStandardSupport.java]: in all methods there must be space after close bracket  `) {`
Fix all cases in support method. 

NB! In array definition `){` is OK

**Comment by malinink** [comment]
@janchk Still awaiting for fixes...

**Comment by janchk** [comment]
@malinink I'll do it today.

**Comment by janchk** [comment]
@malinink all bugs are fixed.

**Comment by malinink** [comment]
@janchk @Yarik2308 
Take a look at 34ee1209625492c4bed41350941f1daf6f42f555 and 5337ded6d4d83b4f707183723793e9f4b6522079

**Pull request merged**